app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
A useful query. We are aware of that point, although we (I) don't have a good grasp of the other possible reasons for withdrawal.
It shouldn't affect clients currently, and we wanted to avoid introducing a more complex dependency on the details of the publishing-api workflow model.